src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int TYPE_OF_OPERATION_INDEX = 0;
+    private static final int COST_INDEX = 1;
+    private static final String SUPPLY_WORD = "supply";
+    private static final String BUY_WORD = "buy";
+    private static final String RESULT_WORD = "result";
+    private static final String VIRGULE_SYMBOL = ",";
```

Comment:
VIRGULE - it's the French name of the comma. In variable naming, you should use English words 
```suggestion
    private static final String COMMA_SYMBOL = ",";
```